pkg/registry/core/pod/etcd/eviction.go

```diff
@@ -29,6 +30,12 @@ import (
 	"k8s.io/kubernetes/pkg/runtime"
 )
 
+const (
+	// Max size of DisruptedPod size. This equals to the number of evictions handled
```

Comment:
Change the comment to:

Max size of PodDisruptionBudgetStatus.DisruptedPods. API server eviction subresource handler will refuse to evict pods covered by the corresponding PDB if the size of the map exceeds this value. It means a large number of evictions have been approved by the API server but not noticed by the PDB controller yet. This situation should self-correct because the PDB controller removes entries from the map automatically after the PDB DeletionTimeout regardless.